src/reanimated2/NativeReanimated/NativeReanimated.ts

```diff
@@ -10,6 +9,7 @@ import { jsVersion } from '../platform-specific/jsVersion';
 import type { WorkletRuntime } from '../runtimes';
 import { getValueUnpackerCode } from '../valueUnpacker';
 import type { LayoutAnimationBatchItem } from '../layoutReanimation/animationBuilder/commonTypes';
+import NativeReanimatedModule from '../../specs/NativeReanimatedModule';
```

Comment:
Actually, `Native` prefix in `NativeReanimatedModule.ts` is just for react-native-codegen to detect this file. The actual name of the module is `ReanimatedModule` so maybe let's import it under that name?

Also, `NativeReanimatedModule` already represents something else in our codebase so this ambiguity might be confusing.

```suggestion
import ReanimatedModule from '../../specs/NativeReanimatedModule';
```